Bottom: 4c69a188a96268f07c5d7461565f8d0bf08ae799
Top:    a4b027aa49511f9b25d3ae96dfdd17cf88185db2
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2014-04-16 23:55:13 -0700

Refresh of rs-ctrl

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 3af60c4..3f9fe49 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -862,7 +862,7 @@ static int rs_create_ep(struct rsocket *rs)
 	if (ret)
 		return ret;
 
-	rs->sq_inline = qp_attr->cap.max_inline_size;
+	rs->sq_inline = qp_attr.cap.max_inline_size;
 	if (rs->sq_inline < RS_MIN_INLINE)
 		return ERR(EINVAL);
